squarelet/core/views.py

```diff
@@ -19,6 +21,21 @@ class ERHLandingView(TemplateView):
 
     template_name = "core/erh_landing.html"
 
-    def get_context_date(self, **kwargs):
+    def get_context_data(self, **kwargs):
         context = super().get_context_data()
+        if self.request.user.is_authenticated:
+            context["can_access_hub"] = self.request.user.is_hub_eligible
+            context["eligible_orgs"] = self.request.user.organizations.filter(
+                Q(individual=False) &
+                (Q(hub_eligible=True)
+                 | Q(groups__hub_eligible=True)
+                 | Q(parent__hub_eligible=True))
+            )
+            context["group_orgs"] = self.request.user.organizations.filter(individual=False)
+            if self.request.user.is_hub_eligible:
+                # TODO cache these context values so that we're not making
+                # a roundtrip to Airtable every time we render the template
```

Comment:
@mitchelljkotler Here is where caching would really come in handy. Even during development, I'm waiting a few seconds for the resources to load in each time I refresh the page.